dev-packages/node-integration-tests/suites/tracing/apollo-graphql/test.ts

```diff
@@ -3,13 +3,13 @@ import { createRunner } from '../../../utils/runner';
 
 // Graphql Instrumentation emits some spans by default on server start
 const EXPECTED_START_SERVER_TRANSACTION = {
-  transaction: 'Test Server Start',
+  transaction: 'Test Server Start (query IntrospectionQuery)',
 };
 
 describe('GraphQL/Apollo Tests', () => {
   test('should instrument GraphQL queries used from Apollo Server.', async () => {
     const EXPECTED_TRANSACTION = {
-      transaction: 'Test Transaction',
+      transaction: 'Test Transaction (query)',
```

Comment:
is this lacking a query name on purpose? 🤔

Reply:
The query in this test doesn't have a name:
```ts
      await server.executeOperation({
        query: '{hello}',
      });
 ```
 
 Whereas, when it has one like this:
 ```ts
     await apolloServer.executeOperation({
      query: 'query GetHello {hello}',
    });
 ```
 the transaction is named like this: Test Transaction (query GetHello)

Reply:
ahh right, makes sense 👍